internal/source/containers_image_test.go

```diff
@@ -396,7 +396,7 @@ func TestImageRegistry(t *testing.T) {
 				// If the digest should already exist check that we actually hit it
 				if tt.digestAlreadyExists {
 					assert.Contains(t, buf.String(), "image already unpacked")
-					assert.Equal(t, rs.UnpackTime, unpackDirStat.ModTime())
+					assert.Equal(t, rs.UnpackTime, unpackDirStat.ModTime().Truncate(time.Second))
```

Comment:
I think even asserting on the second could be brittle if we happen to run on the second to next second break. How about just asserting the duration is in a reasonable range?
```suggestion
					assert.WithinDuration(t, rs.UnpackTime, unpackDirStat.ModTime(), time.Second)
```

Reply:
This one should be exact. `rs.UnpackTime` is always truncated to the second in the same way.  I'll add a comment in the code about why we need that though.